renovate.json

```diff
@@ -11,6 +11,17 @@
     ":autodetectPinVersions"
   ],
   "ignorePaths": [".kokoro/requirements.txt"],
+  "customManagers": [
+    {
+      "customType": "regex",
+      "fileMatch": [
+        "^.kokoro/presubmit/graalvm-native.*.cfg$"
+      ],
+      "matchStrings": ["value: \"gcr.io/cloud-devrel-public-resources/graalvm.*:(?<currentValue>.*?)\""],
+      "depNameTemplate": "com.google.cloud:google-cloud-shared-config",
```

Comment:
Does this repo know about `google-cloud-shared-config`? Or should this be `native-image-shared-config`?

Reply:
Good catch. For consistency, it makes sense for this to use native-image-shared-config.